Book-A-Desk.Api.Tests/FilteredBookingsTest.fs

```diff
@@ -0,0 +1,90 @@
+module Book_A_Desk.Api.Tests.FilteredBookingsTest
+
+open FsToolkit.ErrorHandling
+open System
+open Xunit
+
+open Book_A_Desk.Api.Models
+open Book_A_Desk.Domain.Office.Domain
+open Book_A_Desk.Infrastructure.DynamoDbEventStore
+open Book_A_Desk.Domain
+open Book_A_Desk.Infrastructure
+
+open Book_A_Desk.Domain.Events
+open Book_A_Desk.Domain.Reservation
+open Book_A_Desk.Domain.Reservation.Queries
+
+open Book_A_Desk.Domain.Reservation.Events
+
```

Comment:
A general tidyness comment.

Having a tidy codebase is a pretty important thing. It helps readability and understanding but also prevents the Broken Window Theory. Essentially, if a codebase looks unkempt, it's very easy to let the quality of the code go since if nobody put effort into maintaining it, why should you?

I don't mean that your code isn't clean (it's generally pretty good), but there are some patterns that I see that should be fixed. These can sound like nitpicks or very minor details, but I think they do make a difference in the long-term health of the codebase.

Here's a few things that I see:
1. You generally want to keep a maximum of 1 empty line in-between lines of code (ex: lines 18 and 19, lines 35 and 36 violate this rule).
2. If you are specifying the type of a variable or function, please keep one empty space behind and after `:`. Some examples: line 23, line 29, line 52, line 60. So `let createDeskBookedEvent date officeId email : DomainEvent`. This is a F# standard, so it's okay if you're not familiar since you are learning the language.

Reply:
To reiterate: I don't think your code is not tidy. Generally, it is pretty good. I wrote this long text because I want to explain why I added these comments when they can look like meaningless nitpicks. I'd rather overexplain something you might know already then underexplain and have you misunderstand my message :).